<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QDialog" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(0, 0, 0);
}

QMainWindow{
background-color: rgb(0,0,0);
}</string>
  </property>
  <widget class="QLabel" name="txtSignupTitle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Naskh Arabic Semi Bold</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpFullName">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>241</y>
     <width>301</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: white;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpAge">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>287</y>
     <width>301</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpPasswordR">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>379</y>
     <width>301</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txtBankID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>316</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="text">
    <string>Create Bank ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtPasswordR">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>362</y>
     <width>104</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(195, 195, 195);
background-color: none;
}</string>
   </property>
   <property name="text">
    <string>Create Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpBankId">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>333</y>
     <width>301</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txtAge">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="text">
    <string>Enter Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtFullName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>224</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="text">
    <string>Enter Full name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBackR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
}</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
	border: none;
border-bottom: 2px solid cyan;
border-right: 2px solid cyan;
border-radius: 6px;

}

QMessageBox {
color: white;
background-color: black;
}</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
